concourse/model/traits/component_descriptor.py

```diff
@@ -83,6 +85,12 @@
             If not configured, the CICD-landscape's default ctx will be used.
         '''
     ),
+    AttributeSpec.optional(
+        name='component_descriptor_labels',
```

Comment:
I would prefer to call it `component_labels` - it is already below the `component_descriptor_trait` (DRY)